pgmq/extension/src/lib.rs

```diff
@@ -34,11 +34,13 @@ fn pgmq_create(queue_name: &str) -> Result<(), PgmqExtError> {
 }
 
 #[pg_extern]
-fn pgmq_create_partitioned(
+fn pgmq_create(
     queue_name: &str,
-    partition_size: default!(i64, 10000),
+    partition_interval: default!(String, "'daily'"),
+    retention_interval: default!(String, "'5 days'"),
```

Comment:
default is a new partition daily, with 5 day retention of the partitions. Will need to add some better docs around this, because its easy to confuse this with message retention.